source/classes.tex

```diff
@@ -3317,7 +3317,8 @@
 member functions.
 
 \pnum
-A union for which objects, pointers, or references are declared is not an anonymous union.
+\begin{note}
```

Comment:
Regarding objects vs. pointers: I think we should keep this as-is, because "for which ... are declared" is a bit hazy. What does "for which" mean, exactly? So, let's keep the hand-waving. In particular since this is a note now, anyway.